usecases/continuous_screening/common.go

```diff
@@ -0,0 +1,5 @@
+package continuous_screening
+
+func counterpartyIdentifier(objectType, objectId string) string {
```

Comment:
```suggestion
func typedObjectId(objectType, objectId string) string {
```
Or similar. The original name only made sense in the transaction monitoring context.